presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -169,6 +169,7 @@
     public static final String MAX_TASKS_PER_STAGE = "max_tasks_per_stage";
     public static final String DEFAULT_FILTER_FACTOR_ENABLED = "default_filter_factor_enabled";
     public static final String PUSH_LIMIT_THROUGH_OUTER_JOIN = "push_limit_through_outer_join";
+    public static final String OPTIMIZE_CONSTANT_GROUPING_KEYS = "optimize_constant_grouping_keys";
```

Comment:
Maybe we don't need this option as this is always a good thing?

Reply:
it's helpful to debug if something goes wrong (I had some bugs in the code and changing the config helped me find them quicker)